### ISSUE TITLE
Attaching debuggers to the assignment code

### ISSUE THREAD

**Issue by AtheMathmo** [dilevin/CSC417-physics-based-animation#1]
Could you please ELI5 how to get the assignment code to run with a debugger. Please save me from `std::cout`...

A solution involving VSCode would be welcome :)

**Comment by psarahdactyl** [comment]
I use `lldb`. I run `lldb ./a4-cloth` for example which starts the debugger and loads the debug symbols for your executable. (For this reason, you must have compiled with `-DCMAKE_BUILD_TYPE=Debug`. Then I use `run` to run the executable and after it has crashed, I run `bt` which gives me the backtrace/call stack so I can see in which function the whole thing went wrong. And that's literally all I know about it. I'm sure some people in the class know more than I do about this, but it's enough to get out of writing print statements everywhere.

**Comment by AtheMathmo** [comment]
I forgot to reply to this before.

This is super helpful, thank you!!
